assets/css/style.css

```diff
@@ -0,0 +1,140 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+/* the class of the header  */
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.header h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.header h1 .seo {
+  color: #d9dcd6;
+}
+
+.header nav {
+  padding-top: 15px;
+  margin-right: 20px;
+  float: right;
+  font-family: "Gill Sans", "Gill Sans MT", "Calibre", "Trebuchet MS",
+    sans-serif;
+  font-size: 20px;
+}
+
+/* the links from the navbar */
+.header nav ul li {
+  display: inline-block;
+  margin-left: 25px;
+  list-style-type: none;
+}
+
+a {
+  color: #ffffff;
+  text-decoration: none;
+  font-size: 16px;
+}
+
+p {
+  font-size: 16px;
+}
+
+/* the image  */
+.hero {
+  height: 800px;
+  width: 100%;
+  margin-bottom: 25px;
+  background-image: url("../images/digital-marketing-meeting.jpg");
+  background-size: cover;
+  background-position: center;
+}
+
+.float-left {
+  float: left;
+  margin-right: 25px;
+}
+
+.float-right {
+  float: right;
+  margin-left: 25px;
+}
+.content {
+  width: 75%;
+  display: inline-block;
+  margin-left: 20px;
+}
+/* the aside classes */
+
+.benefits {
+  margin-right: 20px;
+  padding: 20px;
+  clear: both;
+  float: right;
+  width: 20%;
+  height: 100%;
+  font-family: "Gill Sans", "Gill Sans MT", "Calibre", "Trebuchet MS",
+    sans-serif;
+  background-color: #2589bd;
+}
+
+/* class of the sections from the aside */
+.benefit-cost {
+  margin-bottom: 32px;
+  color: #ffffff;
+}
+
+.benefit-cost h3 {
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.benefit-cost img {
+  display: block;
+  margin: 10px auto;
+  max-width: 150px;
+}
+/* classes of the sections from the  main */
```

Comment:
This `main` elements should go above the elements for the `aside` because they come first in the HTML file.

Reply:
Thanks Natasha - for the note and for your comments on the grading feedback :). I've updated the CSS commentary.